extensions/color-picker/package.json

```
@@ -76,6 +76,12 @@
       "name": "convert-color",
       "title": "Convert Color",
       "description": "Convert any color to a different format.",
+      "mode": "no-view"
+    },
+    {
+      "name": "name-color",
+      "title": "Name Color",
+      "description": "name the color selected.",
```

I would suggest a similar naming than the `Pick Color` command:
```suggestion
      "description": "Name a color on your desktop",
```

---

fixed, thanks!